packages/fluent-theme/src/fluent/styles/SearchBox.styles.ts

```diff
@@ -1,14 +1,43 @@
 import { ISearchBoxStyleProps, ISearchBoxStyles } from 'office-ui-fabric-react/lib/SearchBox';
 
 export const SearchBoxStyles = (props: ISearchBoxStyleProps): Partial<ISearchBoxStyles> => {
-  const { theme } = props;
+  const { theme, underlined } = props;
   const { effects, palette } = theme;
 
   return {
     root: [
       {
         borderRadius: effects.roundedCorner2,
-        borderColor: palette.neutralSecondaryAlt
+        borderColor: palette.neutralTertiary
+      },
+      underlined && [
```

Comment:
no need to wrap this object with square brackets. This way you make the life easier for merge-styles parsing phase 😄